vite.config.js

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
```

Comment:
What's a plugin?

Reply:
so vite by itself doesnt natively support vue, so they had made a package for vite so that it can render the vue file.
which in this case is the @vitejs/plugin-vue.
https://vitejs.dev/guide/api-plugin.html